### DISCUSSION TITLE
Is anything required to make this start on computer reboot?

### DISCUSSION THREAD

**Discussion by jctull** [maxarndt/homeassistant-lima#1] [Q&A] [answered]
I was wondering if there is a way to make this launch at startup. I have not used lima, so thought I would ask before searching for an answer elsewhere. Thanks!

**Comment by maxarndt** [comment, marked as answer]
Hey @jctull! Yes, there are multiple ways to launch a Lima VM at startup. 

`Automator.app` can be used to execute a Shell-Script which contains the `limactl start NAME` command (see [this SO answer as a reference](https://stackoverflow.com/a/6445525)).

Alternatively, `launchd` property list can be used to execute the `limactl start NAME` command (see [docs](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html) or [this SO answer as a reference](https://stackoverflow.com/a/13372744)).
